the-blue-alliance-ios/View Controllers/Match/MatchViewController.swift

```diff
@@ -39,18 +42,35 @@ class MatchViewController: MyTBAContainerViewController {
             tbaKit: tbaKit,
             userDefaults: userDefaults
         )
+        
+        infoViewController.matchSummaryDelegate = self
     }
 
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-
+    
     // MARK: - View Lifecycle
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
         Analytics.logEvent("match", parameters: ["match": match.key!])
     }
+    
+}
 
+extension MatchViewController: MatchSummaryViewDelegate {
+    
+    func teamPressed(teamNumber: Int) {
+        guard let event = match.event,
+            let teamKeys = match.teamKeys else { return }
```

Comment:
nit: This should be able to be one line, right? Length doesn't seem like an issue here